### PR TITLE
Bug 1870076: Gherkin Scripts for ODC-4004

### PR DESCRIPTION
**Pull request by gajanan-more**
Fixes: https://issues.redhat.com/browse/ODC-4256

Problem: Gherkin Scripts Design for Knative Eventing Broker and Trigger epic ODC-4004

Solution: These are the gherkin scripts designed covering all the scenarios related to Knative Eventing Broker and Trigger epic ODC-4004

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/Knative/broker-ODC-4004.feature [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+@regression
+Scenario: Edit Labels action on Broker
+    Given user is at Developer Perspective
+    And user is having Broker on the Topology page
+    When user right clicks on the Broker to open the context menu
+    And user clicks on the Edit Labels
+    And user adds the label
+    And user clicks on the Save button on the modal to save labels and close the modal
+    And user clicks on the Broker to open the sidebar
+    And user opens   the Details tab
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/Knative/broker-ODC-4004.feature:98]: Extra space after opens

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/Knative/broker-ODC-4004.feature]: Updated.

**Pull request merged**